### PR TITLE
initial map setup

### PR DESCRIPTION
**Pull request by tonypls**
## Issue
Setup map 

Should the MapStyle.json live in the map directory? 
Do we want camel casing for all files even json?

### CODE REVIEW

**Comment by madsnedergaard** [comment]
> Should the MapStyle.json live in the map directory? 

Hmm, it could also live in the assets/public folder instead?

> Do we want camel casing for all files even json?
Good question 🤔 
I don't have any specific preferences here - we could also go `kebab-case` for non-components and `PascalCase` for components? Doesn't seem like we can set up a rule for differentiating though.

https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/filename-case.md

**Comment by VIKTORVAV99** [comment]
We could do it if we name them something like `<name>.component.tsx` and add eslint overrides for `*.component.tsx`. It would make things very clear but create long file names so there are both pros and cons.

**Comment by tonypls** [comment]
I think this is a good approach and easy to remember,[here]( https://basarat.gitbook.io/typescript/styleguide#filename)

> Filename
Name files with camelCase. E.g. utils.ts, map.ts etc.
Reason: Conventional across many JS teams.
When the file exports a component and your framework (like React) wants component to be PascalCased, use pascal case file name to match e.g. Accordion.tsx, MyControl.tsx.
Reason: Helps with consistency (little overthought required) and its what the ecosystem is doing.

I love kebabs but might be better to follow the more common pattern

File: web/package.json [hunk 1]
```diff
@@ -31,8 +31,11 @@
   "dependencies": {
     "@tanstack/react-query": "4.13.0",
     "@tanstack/react-query-devtools": "^4.13.0",
+    "mapbox-gl": "npm:empty-npm-package@1.0.0",
```
Comment (VIKTORVAV99) [inline web/package.json:34]: Just curious about this one.
Is this just temporary or how does it work and why do we need it?

Comment (tonypls) [reply web/package.json:34]: So I think this is a required dep for react-map-gl but map-libre-gl does everything it was required for

Comment (tonypls) [reply web/package.json:34]: It is a bit curious, maybe I try delete it or add a comment explaining

Comment (VIKTORVAV99) [reply web/package.json:34]: Ah it's for the peer dependency, that makes sense, I have never seen it used before tho. 👍🏻

Comment (VIKTORVAV99) [reply web/package.json:34]: Linking this should be enough? https://visgl.github.io/react-map-gl/docs/get-started/get-started#using-with-a-mapbox-gl-fork

**Comment by madsnedergaard** [comment]
> I think this is a good approach and easy to remember,[here](https://basarat.gitbook.io/typescript/styleguide#filename)
> 
> > Filename
> > Name files with camelCase. E.g. utils.ts, map.ts etc.
> > Reason: Conventional across many JS teams.
> > When the file exports a component and your framework (like React) wants component to be PascalCased, use pascal case file name to match e.g. Accordion.tsx, MyControl.tsx.
> > Reason: Helps with consistency (little overthought required) and its what the ecosystem is doing.
> 
> I love kebabs but might be better to follow the more common pattern

Sounds good to me :)
I also think `xx.component.tsx` will be too long (and not working with VSCode refactoring functionality). If we want to separate things in linting, we could probably also rely on `.ts` vs `.tsx` later on.

**Review by madsnedergaard — APPROVED** [review]
LGTM, we will most likely delete the MapStyle.json file later on anyway (when our geo-tooling is in place)

**Pull request merged**